experimental/PropertyDDS/packages/property-binder/jest.config.js

```diff
@@ -11,4 +11,8 @@ module.exports = {
 
 	// An array of regexp pattern strings that are matched against all test paths, matched tests are skipped
 	testPathIgnorePatterns: ["/node_modules/"],
+
+	// Force the CJS entry point for 'msgpackr'.  (Problem still present in msgpackr@1.9.7).
+	// See: https://stackoverflow.com/questions/73203367/jest-syntaxerror-unexpected-token-export-with-uuid-library
+	moduleNameMapper: { "^msgpackr$": "msgpackr" },
```

Comment:
'msgpackr' has the same entry point issue as 'uuid'.  Unfortunately, the latest version does not contain a fix.  Forcing CJS is the only option.